src/backend/distributed/sql/udfs/isolate_shards/latest.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TYPE citus.shard_isolation_mode AS ENUM (
```

Comment:
name of the directory should match the function name (isolate_tenant_to_new_shard)